src/main/java/io/openliberty/tools/common/plugins/util/BinaryScannerUtil.java

```diff
@@ -162,6 +132,48 @@ public BinaryScannerUtil(File binaryScanner) {
         return featureList;
     }
 
+    public Set<String> reRunBinaryScanner(Set<String> allClassesDirectories, String eeVersion, String mpVersion)
```

Comment:
Perhaps a comment here to indicate how this run differs from `runBinaryScanner()`

Reply:
ok